### PR TITLE
Add U9 release blog

### PR DESCRIPTION
**Pull request by chathurace**
Add U9 release blog

### CODE REVIEW

File: _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+As an integration language, data security is a main priority of Ballerina. Therefore, in order to overcome possible weaknesses of traditional crypto algorithms against quantum computing, Ballerina adds support for quantum-safe crypto algorithms from this release. With this functionality, Ballerina programs can now use crypto algorithms such as ML-KEM-768, RSA-KEM-ML-KEM-768, and ML-DSA65 to encrypt and sign data.
+
+See Ballerina <a href="https://central.ballerina.io/ballerina/crypto/latest" target="_blank">crypto package</a> for more information.
+
+## Improved JSON and XML handling
+
+With this release, it is possible to navigate JSON data in Ballerina using JSONPath expressions. Complex data retrievals can be performed within the Ballerina code using this feature as follows:
+
+```
```
Comment (sm1990) [inline _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md:33]:
```suggestion
```ballerina
```

**Comment by sm1990** [comment]
LGTM

File: _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+See <a href="https://ballerina.io/learn/persist-cli-tool/#generate-the-data-model-by-introspecting-an-existing-database-[experimental]" target="_blank">database introspection</a> for more information.
+
+See the <a href="https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.9.0" target="_blank">release note</a> for a comprehensive overview of Swan Lake Update 9's new features and improvements. We encourage our community to explore these features and provide feedback. Your input is invaluable in shaping the future of Ballerina and ensuring it meets your needs.
+
+Cheers to the Ballerina community and the bright future of it!
+
+
+
```
Comment (anupama-pathirage) [inline _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md:85]: Extra new lines

File: _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+io:println(sum);
+```
+
+In addition, the data projection feature enables the conversion of JSON or XML data into Ballerina records by specifying only the required fields from the source data. This is useful for extracting only the required information from very large JSON or XML messages, which are common in interactions with B2B and enterprise applications.
+
+See <a href="https://ballerina.io/learn/by-example/jsonpath-expressions/" target="_blank">JSONPath</a> and <a href="https://ballerina.io/learn/by-example/json-to-record-with-projection/" target="_blank">JSON data projection</a> examples.
+
+## SOAP support
+
+This release introduces support for invoking SOAP 1.1 and 1.2 endpoints with web services security policies. This enables Ballerina programs to communicate securely with SOAP services, and for example, expose SOAP services as REST endpoints.
```
Comment (anupama-pathirage) [inline _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md:54]:
```suggestion
This release introduces support for invoking SOAP 1.1 and 1.2 endpoints with web services security policies. This enables Ballerina programs to communicate securely with SOAP services and, for example, expose SOAP services as REST endpoints.
```

File: _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+The problem with this approach is that developers either have to store the generated code in a version control system, or they have to manually generate code each time before a build. With this release, Ballerina developers can overcome this problem by using the code generation capabilities integrated with the build process. Information required for code generation can be specified in the `Ballerina.toml` file. For example, the following configuration will generate a REST client and Ballerina records for the OpenAPI schema given in a file named `openapi.yaml` in the current directory.
+
+```toml
+[[tool.openapi]]
+id = "client"
+filePath = "./openapi.yaml"
+targetModule = "delivery01"
+options.mode = "client"
+```
```
Comment (anupama-pathirage) [inline _posts/2024-05-27-announcing-ballerina-2201.9.0-swan-lake-update-9.md:70]: @chathurace As per your comment on the doc, we thought of removing this no?

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**